public/fonts/InstrumentSans/font.css

```diff
@@ -0,0 +1,40 @@
+/* Instrument Sans font family */
+
+@font-face {
+  font-family: 'Instrument Sans';
+  font-style: normal;
+  font-display: swap;
+  src: local('Instrument Sans'), url('./InstrumentSans[wdth\,wght].woff2') format('woff2');
+}
+
+/* @font-face {
+  font-family: 'Instrument Sans';
+  font-weight: 400;
+  font-style: normal;
+  font-display: swap;
+  src: local('Instrument Sans'), url('./InstrumentSans-Regular.woff2') format('woff2');
+}
+
+@font-face {
+  font-family: 'Instrument Sans';
+  font-weight: 500;
+  font-style: normal;
+  font-display: swap;
+  src: local('Instrument Sans'), url('./InstrumentSans-Medium.woff2') format('woff2');
+}
+
+@font-face {
+  font-family: 'Instrument Sans';
+  font-weight: 600;
+  font-style: normal;
+  font-display: swap;
+  src: local('Instrument Sans'), url('./InstrumentSans-SemiBold.woff2') format('woff2');
+}
+
+@font-face {
+  font-family: 'Instrument Sans';
+  font-weight: 700;
+  font-style: normal;
+  font-display: swap;
+  src: local('Instrument Sans'), url('./InstrumentSans-Bold.woff2') format('woff2');
+} */
```

Comment:
Is this commented code intentional? Why?

Reply:
Just in case we need to use the static version of the font instead of the variable version it is already there, so we can switch quickly (same with the font files, I've uploaded both variable and static font versions).